tests/e2e/verification_post_day1_test.go

```diff
@@ -473,4 +473,37 @@ var _ = Describe("Verify cluster", func() {
 			Expect(clusterResp.Body().MultiArchEnabled()).To(BeFalse())
 		}
 	})
+
+	It("imdsv2 is set correctly - [id:75372]", ci.Day1Post, ci.Critical, func() {
```

Comment:
Can we merge this one with vase 63950 ?

Reply:
Can't we keep in 75372 only as for hcp later it will be made required as default and we have to check nodepool as well , not in classic cluster

Reply:
ok. fair enough

Reply:
Thanks